src/praisonai/praisonai/cli/commands/managed.py

```
@@ -397,6 +422,29 @@ def agents_update(
+    Example:
+        praisonai managed agents delete agent_01AbCdEf
+        praisonai managed agents delete agent_01AbCdEf --yes
+    """
+    if not confirm:
+        typer.confirm(f"Delete agent {agent_id}? This cannot be undone.", abort=True)
+    
+    try:
+        client = _get_client()
+        client.beta.agents.delete(agent_id)
+        typer.echo(f"Agent {agent_id} deleted successfully")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)